dojo/api_v2/views.py

```diff
@@ -106,6 +108,14 @@ class DojoGroupViewSet(prefetch.PrefetchListMixin,
     def get_queryset(self):
         return get_authorized_groups(Permissions.Group_View).distinct()
 
+    def perform_create(self, serializer):
+        group = serializer.save()
+        group_post_create(group)
+
+    def perform_destroy(self, instance):
+        instance.delete()
+        group_post_delete(instance)
+
```

Comment:
Did you consider django signals as a way to implement post_create?

Reply:
Good idea, I will check

Reply:
Or overriding the `.save()` method

Reply:
changed to signals